iommu_in_memory_queues.adoc

```diff
@@ -414,6 +461,12 @@ guaranteed to not complete before all previously fetched commands were executed
 and completed. A previously fetched ATS command to invalidate device ATC does 
 not complete till either the request times out or a valid response is received
 from the device.
+
+If one or more ATS invalidation commands preceeding the `IOFENCE.C` have timed
+out, then software may reset the CQ and resubmit the invalidation commands that
```

Comment:
Not sure what you mean here by reset. I assume it means set again and not actually trigger a reset.

Reply:
I will reword to state, "make the CQ operational and ..."

Reply:
updated.